code/game/turfs/simulated/floor/plating.dm

```diff
@@ -140,8 +140,8 @@
 
 /turf/simulated/floor/engine/cult/New()
 	..()
-	if(ticker.mode)//only do this if the round is going..otherwise..fucking asteroid..
-		icon_state = ticker.mode.cultdat.cult_floor_icon_state
+	if(ticker)//only do this if the round is going..otherwise..fucking asteroid..
```

Comment:
say "we have a fire cult because the asteroid turfs are fire instead of normal" :|